### PR TITLE
(maint) move to use the arm64 pool

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by Bdsjewelry** [comment]
`{"commit":"5179c75b0e98677fcb5229213775e62496446774","error":"yaml: missing kind attribute","event":"push","level":"warning","msg":"trigger: cannot parse yaml","ref":"refs/heads/master","repo":"Arch/aur-builds","time":"2022-09-17T22:30:33Z"}`

File: .drone.yml [hunk 1]
```diff
@@ -139,4 +104,4 @@ trigger:
 
 depends_on:
 - linux-arm64
-# - linux-arm
+
```
Comment (Bdsjewelry) [inline .drone.yml:107]: tphoney:arm64_build

Comment (tphoney) [reply .drone.yml:107]: I am not sure what you are trying to tell me ?
the drone.yml is valid and builds are running.

**Comment by Bdsjewelry** [comment]
Hi I am Sorry I wasn't trying to say anything I am new at this and I'm just
trying to learn. I apologize and I hope you understand.

**Pull request merged**